plugin/base.go

```diff
@@ -66,3 +73,21 @@ func getRegion() string {
 func getMemorySize() int {
 	return lambdacontext.MemoryLimitInMB
 }
+
+func getLogGroupName() string {
+	return lambdacontext.LogGroupName
+}
+
+func getLogStreamName() string {
+	return lambdacontext.LogStreamName
+}
+
+// GetFromContext returns InvokedFunctionArn and AwsRequestID if available.
+func GetFromContext(ctx context.Context) (string, string) {
```

Comment:
Function name is to generic even though it provides very specific properties. You can define two different functions (`getInvokedFunctionArn` and `getAwsRequestID`). In this case, `lambdacontext.FromContext(ctx)` will be called two times but it is not a problem.